### DISCUSSION TITLE
BuiltIn Taxonomy for groups/password storage

### DISCUSSION THREAD

**Discussion by MartinParis** [keepassxreboot/keepassxc#10483] [Q&A]
## Summary
Is there a possible generic taxonomy for all groups in the database, as a feature when creating a new database?

## Context
After over 10 years and 500+ plus password entries in my database, it is a mess.
Passwords are spread over Groups, in the wrong Groups, etc.

I've started a few times to organize better, but always struggled with a logical taxonomy for the Groups.
Searching for guidance on Internet, when I search anyway, does not help me towards that goal.
So I am wondering if you, the gods, can come forth with a taxonomy, or at least the 10 Commandments on making choices for the taxonomy. Did you already do that and kept it hidden? (a person can hope...).

A built in version would fullfill my dreams.....

**Comment by droidmonkey** [comment]
Can you elaborate more on what you are looking for? Organizing your database is truly up to you as the end user. It all depends on your use case, workflow, etc. Most of our tools don't care about organization, like auto-type and browser integration.

**Comment by phoerious** [comment]
I have a bunch of groups for different purposes such as local machine passwords, various keys etc. The vast majority of passwords, however, live in a single group called "Web services" and I am using search and auto fill matching for retrieving them. Don't overthink your group structure if you can use search. Too many groups just make finding stuff harder.

**Reply by MartinParis** [reply]
Yes, search works excellent. In some cases however the search does find too many entries.
Maybe I should add those to a few Groups and the rest not.
